tsconfig.json

```diff
@@ -1,20 +1,27 @@
 {
   "compilerOptions": {
-    "experimentalDecorators": true,
+    "composite": true,
+    "emitDeclarationOnly": true,
+    "declarationDir": "./build/types",
```

Comment:
Participating in a composite build requires the project to emit something - can be declaration files only as we now have here.